src/query/api/v1/handler/influxdb/write.go

```diff
@@ -0,0 +1,255 @@
+/*
+ * Author: Markus Stenberg <mstenber@aiven.io>
+ *
+ * Copyright (c) 2019 Aiven Oy
+ */
+
+package influxdb
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	imodels "github.com/influxdata/influxdb/models"
+	"github.com/m3db/m3/src/cmd/services/m3coordinator/ingest"
+	"github.com/m3db/m3/src/dbnode/client"
+	"github.com/m3db/m3/src/query/api/v1/handler"
+	"github.com/m3db/m3/src/query/models"
+	"github.com/m3db/m3/src/query/ts"
+	"github.com/m3db/m3/src/query/util/logging"
+	xerrors "github.com/m3db/m3/src/x/errors"
+	"github.com/m3db/m3/src/x/instrument"
+	xhttp "github.com/m3db/m3/src/x/net/http"
+	xtime "github.com/m3db/m3/src/x/time"
+	"go.uber.org/zap"
+)
+
+const (
+	InfluxWriteURL = handler.RoutePrefixV1 + "/influxdb/write"
```

Comment:
Surprised the linter didn't catch this, since `InfluxWriteURL` it will need a comment. i.e. `// InfluxWriteURL is the Influx DB write handler URL`

Reply:
Yeah... I think our linter hasn't been running for a while (tbh kinda dreading turning it back on since I'm sure there'll be hours of boring renaming involved :p )